src/PSR7Session/Http/SessionMiddleware.php

```diff
@@ -291,9 +301,23 @@ private function getTokenCookie(SessionInterface $sessionContainer) : SetCookie
      */
     private function getExpirationCookie() : SetCookie
     {
+        $currentTimeProvider = $this->currentTimeProvider;
+        $dateTime            = $currentTimeProvider();
```

Comment:
It's a minor thing but we could do something like do have a better name than `$dateTime`:

```php
$expirationDate = $currentTimeProvider()->modify('-30 days');
```